lib/presentation/bill_details/components/plan_table.dart

```diff
@@ -0,0 +1,182 @@
+          style: AppTextStyle.textStyle.copyWith(
+            fontSize: 24,
+            fontWeight: FontWeight.bold,
+          ),
+        ),
+        const Gap(16),
+        Table(
+          defaultVerticalAlignment: TableCellVerticalAlignment.bottom,
+          columnWidths: const {
+            0: FlexColumnWidth(),
+            1: FlexColumnWidth(2),
+          },
+          children: [
+            TableRow(
+              children: [
+                ClipRRect(
+                  borderRadius:
+                      const BorderRadius.only(topLeft: Radius.circular(8)),
+                  child: Container(
+                    height: 50,
+                    margin: const EdgeInsets.only(right: 1, bottom: 1),
+                    padding:
+                        const EdgeInsets.symmetric(vertical: 12, horizontal: 8),
+                    color: AppColor.blue50Background,
+                    child: Center(
+                      child: Text(
+                        t.billDetailsPage.pricingPlan.columns.standard,
+                        style: headerStyle,
+                      ),
+                    ),
+                  ),
+                ),
+                ClipRRect(
+                  borderRadius:
+                      const BorderRadius.only(topRight: Radius.circular(8)),
+                  child: Container(
+                    height: 50,
+                    margin: const EdgeInsets.only(left: 1, bottom: 1),
+                    padding:
+                        const EdgeInsets.symmetric(vertical: 12, horizontal: 8),
+                    color: AppColor.yellow200,
+                    child: Center(
+                      child: Text(
+                        t.billDetailsPage.pricingPlan.columns.premium,
+                        style: headerStyle,
+                      ),
+                    ),
+                  ),
+                ),
+              ],
+            ),
+            TableRow(
+              children: [
+                ClipRRect(
+                  borderRadius:
+                      const BorderRadius.only(bottomLeft: Radius.circular(8)),
+                  child: Container(
+                    height: 200,
+                    margin: const EdgeInsets.only(right: 1),
+                    alignment: Alignment.center,
+                    padding: const EdgeInsets.all(16),
+                    color: AppColor.blue50Background,
+                    child: Text(
+                      t.billDetailsPage.pricingPlan.details.free,
+                      style: headerStyle,
+                    ),
+                  ),
+                ),
+                ClipRRect(
+                  borderRadius:
+                      const BorderRadius.only(bottomRight: Radius.circular(8)),
+                  child: Container(
+                    height: 200,
+                    margin: const EdgeInsets.only(left: 1, bottom: 1),
+                    color: AppColor.yellow200,
```

Comment:
好みの問題だけど、俺はネストが深いとみづらくなると思ってるので、なるべく子ウィジェットのプロパティで済ませられることは済ませたい！
ので、ClipRRectを削除して、borderRadiusをContainerに直接適用することもできることを書いておきます！

```suggestion
                    decoration: const BoxDecoration(
                    borderRadius: BorderRadius.only(
                      bottomRight: Radius.circular(8),
                    ),
                    color: AppColor.yellow200,
                  ),
```

Reply:
他のClipRRectウィジェットを使用している部分も同様！

Reply:
ClipRRect使用している場所をBoxDecorationに移管しました！Done!